### DISCUSSION TITLE
megaphone function

### DISCUSSION THREAD

**Discussion by jaayb** [thecodingmachine/workadventure#3781] [Q&A]
Is the megaphone function available in the self-hosted version?  I can't seem to figure out how to enable this function.  Thanks.

**Comment by moufmouf** [comment]
Hey @jaayb ,

Yes, you can use the megaphone in the self-hosted version.
In order to have the megaphone function, you need to properly set up the low-level connection to Jitsi, through the following env vars:

```
JITSI_DOMAIN=
JITSI_XMPP_DOMAIN=
JITSI_MUC_DOMAIN=
```

Then, you need to enable the map editor (another env var). When you have access to the map editor from within WorkAdventure available, the megaphone feature is in the map editor config panel.
